libs/external-router/guides/installation.md

```diff
@@ -0,0 +1,7 @@
+# `@daffodil/external-router` Installation
+
+```bash
+npm install @daffodil/external-router --save
+```
+
+Refer to the usage guide for information on how to use `@daffodil/external-router`.
```

Comment:
I think this should say "Refer to the usage guides", since there is no single readme named "usage".

Reply:
I added a TODO. Once this is published to npm, I'll write up a Stackblitz.

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a